cli/args/flags.rs

```diff
@@ -517,6 +517,7 @@ pub struct Flags {
   pub unstable_config: UnstableConfig,
   pub unsafely_ignore_certificate_errors: Option<Vec<String>>,
   pub v8_flags: Vec<String>,
+  pub code_cache_enabled: bool,
```

Comment:
Unfortunately big test change will still be needed in this file because we're only enabling code cache for `run` command initially.

Reply:
I ended up keeping this the same. With the negative `no_code_cache` flag there are actually more test changes needed (because code cache is disabled for non-run commands).